.github/workflows/test_conda-build.yml

```diff
@@ -18,19 +19,27 @@ jobs:
 
```

Comment:
May want to use `actions/setup-python`?

Reply:
Don't think so, this action doesn't use Python?